config/routes.rb

```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
- resources :users
+  resources :users, only: [:index, :new, :create, :edit, :update]
```

Comment:
By the way, you don't need to change anything here. Just sharing this tip with you. There is 7 operation in rails. index, new, create, show, edit, update, destroy. When we need all of these then we use simple `resources` and the name of the controller. If we need only couple of them as a route then we use `only:`. If we saw that we use the maximum of those as routs except one or two items from controller then we can use `except:` . So we can also write this like this ` resources :users, except: [:show, :destroy]`. You will get it [here](https://guides.rubyonrails.org/routing.html#restricting-the-routes-created).